codec-http/src/main/java/io/netty/handler/codec/http/HttpContentEncoder.java

```diff
@@ -205,6 +208,25 @@ protected void encode(ChannelHandlerContext ctx, HttpObject msg, List<Object> ou
         }
     }
 
+    private void encodeFullResponse(HttpResponse newRes, HttpContent content, List<Object> out) {
+        int existingMessages = out.size();
+        assert encodeContent(content, out);
```

Comment:
We don't want this to be an `assert`, right? `assert` statements can be disabled via command line and optimized away by JIT.